lib/state/app_state.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'app_state.freezed.dart';
+
+part 'app_state.g.dart';
+
+@freezed
+class AppState with _$AppState {
+  factory AppState({
+    @Default('') @JsonKey(ignore: true) String name,
```

Comment:
kindly add a todo comment that this is just for testing purposes and and will later be removed once state is finalized 🙂